scripts/lc/lcats

```diff
@@ -506,16 +587,21 @@ elif 'glory' in SYS_TYPE:
 elif 'blue' in SYS_TYPE:
     machineSettings = blueosSettings
 
-elif 'toss' in SYS_TYPE:
-    if 'ruby' in LCSCHEDCLUSTER or 'rzgenie' in LCSCHEDCLUSTER or 'jadedev' == LCSCHEDCLUSTER or 'zindev' == LCSCHEDCLUSTER:
+elif 'toss_3' in SYS_TYPE:
+    if 'rzgenie' in LCSCHEDCLUSTER or 'jadedev' == LCSCHEDCLUSTER or 'zindev' == LCSCHEDCLUSTER:
```

Comment:
should rzgenie be in the toss_4 condition now?

Reply:
rzgenie is running under the TOSS4 conditions now.  This particular branch is only executed if "toss_3" is in the SYS_TYPE, so it's skipped on rzgenie.  Now if for some reason they were to roll rzgenie back to TOSS3 this would activate again, in which case we might still want this check.  Really though we should just cull all the TOSS3 logic in the entire script once we're sure there are no more TOSS3 LC machines.